tests/testthat/test_08_normals.R

```diff
@@ -284,6 +284,37 @@ test_that("normals_dl() downloads normals/frost dates as tibble - multi 1971", {
                   unique(), 0)
 })
 
+test_that("normals_dl() stops if stn argument is provided", {
+  skip_on_cran()
+  skip_if_offline()
+
+  expect_error(normals_dl(stn = "BRANDON A"))
+  expect_error(normals_dl(stn = ""))
+})
+
+test_that("normals_dl() stops if climate normals are not available for stations", {
+  skip_on_cran()
+  skip_if_offline()
+
+  expect_error(normals_dl(climate_ids = c("301AR54", "301B6L0", "301B8LR"),
+                          normals_years = "1971-2000"))
+})
+
+test_that("normals_dl() stops if climate normals are not available some stations", {
+  skip_on_cran()
+  skip_if_offline()
+
+  expect_message(normals_dl(climate_ids = c("301AR54", "2100685", "301B8LR"),
+                          normals_years = "1971-2000"))
+
+  output <- evaluate_promise(normals_dl(climate_ids = c("301AR54", "2100685", "301B8LR"),
+                                        normals_years = "1971-2000"))
+  output_message <- paste(output$messages, collapse = "")
+  climate_ids_message <- sub(".*climate ids: ([^)]+).*", "\\1", output_message)
+  expect_equal(length(strsplit(climate_ids_message, ",")[[1]]), 2)
```

Comment:
I think this part might be a bit overcomplicated (assuming I've understood). 
Does the following achieve the same thing you were aiming for?
```suggestion
  expect_message(normals_dl(climate_ids = c("301AR54", "2100685", "301B8LR"),
                          normals_years = "1971-2000"),
    "Not all stations have climate normals available \\(climate ids: 301AR54, 301B8LR\\)")
```

Reply:
Yup! Mine was definitely an over engineered solution!